client/components/proposal/ProposalVoteStats.tsx

```diff
@@ -1,78 +1,106 @@
-import { ethers } from "ethers";
-import { truncateBalance } from "utils/index";
 import CardGroup from "components/CardGroup";
 import Card from "components/Card";
 import CardLabel from "components/CardLabel";
 import CardStat from "components/CardStat";
 import TokenAmount from "components/TokenAmount";
+import TokenIcon from "components/TokenIcon";
+import DisabledButtonTooltip from "components/DisabledButtonTooltip";
 
 export const ProposalVoteStats = ({
   proposal,
   votePower,
   onVote,
   hasVoted,
+  votingActive,
 }) => {
+  let tooltipText = "";
+  if (!votePower.gt(0)) {
```

Comment:
Would be better to check if it's a BigNumber before checking the value

Reply:
Good catch!